test/shell/t1.sh

```diff
@@ -9,7 +9,13 @@
 set -o errexit
 set -o pipefail
 
-timeout --kill-after=3s 1s ./mgmt --help # hello world!
+if [[ $(uname) == "Darwin" ]] ; then
```

Comment:
Moved the detection for timeout to util/shell. There is a slight issue, with it living in test/util.sh individually test in test/shell/ cannot be run without $timeout being defined.